docs/resources/group.md

```diff
@@ -20,11 +20,11 @@ Groups can contain one or many connectors across different segments. This groupi
 
 ### Required
 
-- **description** (String) The description of the group.
```

Comment:
This file is automatically generated... You need to modify the source file and re-generate the doc.